content/_data/changelogs/lts.yml

```diff
@@ -9639,6 +9620,27 @@
     pr_title: "[JENKINS-71479] Do not use SCSS lighten function directly to avoid invalid CSS"
     message: |-
       Fix invalid CSS which caused some buttons to become invisible on hover (regression in 2.402).
+  lts_changes:
```

Comment:
We only use the `lts-changes` section for the x.1 release.  The x.2 and x.3 are only backports
```suggestion
```